src/components/ProgramRectangles.jsx

```diff
@@ -1,6 +1,4 @@
-// import { ButtonPrimary } from "./buttons/ButtonPrimary";
-
-
+import ButtonPrimary from "./buttons/ButtonPrimary";
 
 export function ProgramLarge() {
```

Comment:
We have pulled the current branch of allprograms. it is different than the code of this pull request here. Can oyu explain the difference between MediumPrograms and LargePrograms

Reply:
ProgramLarge is a left over that is not used anymore